### DISCUSSION TITLE
Update Litemoji to 4.3

### DISCUSSION THREAD

**Discussion by engram-design** [craftcms/cms#12226] [Ideas] [answered]
Would fix https://github.com/verbb/formie/issues/1115#issuecomment-1292697272 where some emoji's are enabled. Tested to be fully compatible with things, and the breaking change from 3.x is PHP 8 support, which Craft 4 should use anyway.

**Comment by brandonkelly** [comment, marked as answer]
Updated for Craft 4.4 via #12234 🎉

**Reply by brandonkelly** [reply]
4.4 is out now!
